### PR TITLE
[Configuration] Add more fine-grained permissions to configurations

### PR DESCRIPTION
**Pull request by dvesh3**
Related to https://github.com/pimcore/data-hub/pull/438

### CODE REVIEW

**Comment by fashxp** [comment]
probably add a migration that re-installs the bundle ... so that already installed bundles don't need to be installed again?

**Comment by dvesh3** [comment]
> probably add a migration that re-installs the bundle ... so that already installed bundles don't need to be installed again?

@fashxp yes, should be handled [here](https://github.com/pimcore/data-importer/pull/122/files#diff-cabfa55630333e0f5e9327612d0d45d3dccec18765ae7d475a6d6d1c3afd3b36R38) as the migration returns true for `checkBundleInstalled`

**Comment by fashxp** [comment]
ahh, oversaw that, perfect 👍

File: src/DependencyInjection/PimcoreDataImporterExtension.php [hunk 1]
```diff
@@ -38,4 +39,19 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.yml');
     }
+
+    /**
+     * @param ContainerBuilder $container
+     */
+    public function prepend(ContainerBuilder $container)
```
Comment (fashxp) [inline src/DependencyInjection/PimcoreDataImporterExtension.php:47]: also need to add `PrependExtensionInterface` in implements statement of class.

**Review by fashxp — CHANGES_REQUESTED** [review]

File: composer.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
   "require": {
     "php": ">=7.4",
     "pimcore/pimcore": "^6.9 || ^10.0",
```
Comment (blankse) [inline composer.json]:
```suggestion
    "pimcore/pimcore": "^10.2",
```

File: composer.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
...
-    "pimcore/data-hub": "^0.7 || ^1.0",
+    "pimcore/data-hub": "^0.7 || ^1.2",
```
Comment (blankse) [inline composer.json]:
```suggestion
    "pimcore/data-hub": "^1.2",
```

File: composer.json [hunk 4]
```diff
@@ -8,7 +8,7 @@
   "require": {
     "php": ">=7.4",
```
Comment (blankse) [inline composer.json]:
```suggestion
    "php": "^8.0",
```

File: src/Resources/public/js/pimcore/configuration/configItemDataObject.js [hunk 5]
```diff
@@ -98,10 +100,6 @@ pimcore.plugin.pimcoreDataImporterBundle.configuration.configItemDataObject = Cl
 
     save: function () {
         //TODO make that more generic in datahub
-        if (!this.isValid(true)) {
```
Comment (fashxp) [inline src/Resources/public/js/pimcore/configuration/configItemDataObject.js]: why was that removed?

**Review by fashxp — CHANGES_REQUESTED** [review]

**Review by fashxp — APPROVED** [review]

**Pull request merged**